src/test/java/api/loans/agetolost/ScheduledAgeToLostFeeChargingApiTest.java

```diff
@@ -0,0 +1,282 @@
+package api.loans.agetolost;
+
+import static api.support.matchers.AccountMatchers.isOpen;
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static api.support.matchers.LoanAccountActionsMatcher.hasLostItemFeeCreatedBySystemAction;
+import static api.support.matchers.LoanAccountActionsMatcher.hasLostItemProcessingFeeCreatedBySystemAction;
+import static api.support.matchers.LoanAccountMatcher.hasLostItemFee;
+import static api.support.matchers.LoanAccountMatcher.hasLostItemFees;
+import static api.support.matchers.LoanAccountMatcher.hasLostItemProcessingFee;
+import static api.support.matchers.LoanAccountMatcher.hasNoLostItemFee;
+import static api.support.matchers.LoanAccountMatcher.hasNoLostItemProcessingFee;
+import static api.support.matchers.ValidationErrorMatchers.hasErrorWith;
+import static api.support.matchers.ValidationErrorMatchers.hasMessage;
+import static java.lang.Boolean.TRUE;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.iterableWithSize;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.folio.circulation.domain.policy.Period;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.hamcrest.Matcher;
+import org.junit.Before;
+import org.junit.Test;
+
+import api.support.builders.FeeFineOwnerBuilder;
+import api.support.builders.ItemBuilder;
+import api.support.builders.ServicePointBuilder;
+import api.support.spring.SpringApiTest;
+import io.vertx.core.json.JsonObject;
+import lombok.val;
+
+public class ScheduledAgeToLostFeeChargingApiTest extends SpringApiTest {
```

Comment:
The name of the class (and my recollection of conversations about this) suggest that the domain terminology for fees is that they are charged (or billed, maybe issued) rather than assigned.

What do you think of changing the names of the tests to reflect that language?

Reply:
Sure, will update tests and code that uses the "assign" term

Reply:
Done, changed to "charge".